### PR TITLE
Fix a few Planemo warnings.

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

File: tools/bedtools/closestBed.xml [hunk 1]
```diff
@@ -55,6 +55,7 @@
                 <option value="a">Report distance with respect to A. When A is on the - strand, "upstream" means B has a higher (start,stop). (-a)</option>
                 <option value="b">Report distance with respect to B. When B is on the - strand, "upstream" means A has a higher (start,stop). (-b)</option>
             </param>
+            <when value="" />
```
Comment (yhoogstrate) [inline tools/bedtools/closestBed.xml]: I am not if this fixes undesired behaviour? (If so, please version++)

Comment (nsoranzo) [inline tools/bedtools/closestBed.xml]: No, it just fixes Planemo (and Galaxy startup) warnings.

Comment (erasche) [inline tools/bedtools/closestBed.xml]: Yep, no change in behaviour. Thanks for fixing the new errors due to added planemo linting @nsoranzo.

**Comment by yhoogstrate** [comment]
:+1: @nsoranzo

**Pull request merged**